README.md

```diff
@@ -1,17 +1,28 @@
 # Crunch
 
+![Crunch logo](crunch.png)
```

Comment:
Seems kinda weird for a fork to use a logo belonging to the original version of the project

Reply:
I had mixed feeling about it too. Especially all I wanted was an image to run crunch on, small (no need more) but with enough details to check for the output. I used a 64px unvanquished icon instead: it is small but detailed enough, and features alpha channel with some shadows, so it looks complex enough to me. It is not displayed, just stored in `test/`.